File: dummy/DTC_BUS_TRACKER/app/src/main/java/com/techakram/dtc_bus_tracker/ui/splash.java
package com.techakram.dtc_bus_tracker.ui;
import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.view.animation.Animation;
import android.view.animation.AnimationUtils;
import android.widget.ImageView;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

import com.techakram.dtc_bus_tracker.MainActivity;
import com.techakram.dtc_bus_tracker.R;

public class splash extends AppCompatActivity {
    ImageView imageView;
    TextView textView;
    Animation top,bottom;
    private  static  int SPLASH_SCREEN =1000;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);
        imageView=findViewById(R.id.image);
         //textView=findViewById(R.id.textview);
        top = AnimationUtils.loadAnimation(this, R.anim.top);
        bottom = AnimationUtils.loadAnimation(this, R.anim.bottom);
        imageView.setAnimation(top);
         //textView.setAnimation(bottom);
        new Handler().postDelayed(new Runnable()
        {
            @Override
            public void run() {
                Intent i = new Intent(splash.this, MainActivity.class);
                startActivity(i);
                finish();
            }
        },SPLASH_SCREEN);
    }
}

File: kotlin Project/TaxiMoh/TaxiMoh_Driver/app/src/main/java/com/uveous/taximohdriver/Splash.kt
package com.uveous.taximohdriver

import android.content.Context
import android.content.Intent
import android.content.SharedPreferences
import android.os.Bundle
import android.os.Handler
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.uveous.loopfoonpay.LoginActivity

class Splash : AppCompatActivity(){

    var id :Int = 0
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.splash)

        startService(Intent(baseContext, MyService::class.java))

        //  getWindow().setNavigationBarColor(getResources().getColor(R.color.navigationbar));
        Handler().postDelayed({
            finish()
            val pref1: SharedPreferences = getSharedPreferences("TASK_ID", Context.MODE_PRIVATE)
            id = pref1.getInt("userid", 0)
            val intent = Intent(this@Splash, LoginActivity::class.java)
            startActivity(intent)
   /*         if (id == 0) {
                val intent = Intent(this@Splash, LoginActivity::class.java)
                startActivity(intent)
            } else{
                val intent = Intent(this@Splash, TravelDashboard::class.java)
                startActivity(intent)
            }*/

        }, 5000)
    }
}